packages/debugger/lib/txlog/actions/index.js

```diff
@@ -211,6 +211,19 @@ export function logEvent(pointer, newPointer, step, decoding, rawEventInfo) {
   };
 }
 
+//this may be replaced once decoding info was added
+export const STORE = "TXLOG_STORE";
+export function store(pointer, newPointer, step, rawSlot, rawValue) {
+  return {
+    type: STORE,
+    pointer,
+    newPointer, //does not actually affect current pointer!
```

Comment:
What do these names mean? Maybe we should use `storagePointer` and `underlyingPointer`?

Reply:
or like source/dest dichotomy

Reply:
Yea, would like to know what these names mean. I was thinking pointers as in linkedlist, or other graph DS.

Reply:
The `pointer` and `newPointer` terminology here is the same as for every other txlog action.  The `pointer` is the JSON pointer to the current location in the txog, and the `newPointer` is the JSON pointer for the new node that is getting created.  They're pointers into the txlog tree; they've got nothing to do with storage pointers.

Likely these terms could be clearer but they're the same as for the other actions; if you think they should be changed, then I would prefer to change them uniformly for all the actions and not just for this one.

Reply:
Makes sense to me. I have to remember the code uses unorthodox pointer names whenever I read this code, and was thrown off when @gnidan interpreted the abstractions differently.